dll/shellext/netplwiz/SHDisconnectNetDrives.c

```diff
@@ -0,0 +1,229 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Disconnect Network Drive
+ * FILE:            dll/shellext/netplwiz/SHDisconnectNetDrives.c
+ * PURPOSE:         Disconnects mapped network drives.
+ * PROGRAMMERS:     Jared Smudde
+ */
+
+#include "netplwiz.h"
+#include <wine/debug.h>
+
+WINE_DEFAULT_DEBUG_CHANNEL(shell);
+
+HINSTANCE hInstance;
+
+VOID InitializeListView(HWND hDlg)
+{
+    HWND hListView = GetDlgItem(hDlg, IDC_CONNECTEDDRIVELIST);
+    LV_COLUMN column;
+
+    if(hListView == NULL)
+        return;
+
+    column.mask = LVCF_WIDTH | LVCF_TEXT;
+    column.pszText = L"Drive Letter";
+    column.cx = 75;
+    ListView_InsertColumn(hListView, 0, &column);
+
+    column.mask = LVCF_WIDTH | LVCF_TEXT;
+    column.pszText = L"Network Path";
+    column.cx = 150;
+    ListView_InsertColumn(hListView, 1, &column);
+}
+
+VOID EnumerateConnectedDrives(HWND hDlg)
+{
+    LV_ITEM item;
+    HIMAGELIST hImgList;
+    HICON hIconDrive = NULL;
+    HMODULE hShell32;
+    HWND hListView = GetDlgItem(hDlg, IDC_CONNECTEDDRIVELIST);
+
+    DWORD dRet;
+    HANDLE hEnum;
+    LPNETRESOURCE lpRes;
+    DWORD dAllocSize = 0x1000;
+    DWORD dCount;
```

Comment:
`d` prefix?
Also, move this next to the `dSize` below.

Reply:
(so it would be instead "dw" prefix).